apps/cowswap-frontend/src/modules/permit/utils/checkIsTokenPermittable.ts

```diff
@@ -88,7 +69,9 @@ export async function estimatePermit(
 
     const gasLimit = estimatedGas.toNumber()
 
-    // TODO: I'm not sure why this is needed, check with Sasha
+    // Sometimes tokens implement the permit interface but don't actually implement it
+    // This check filters out possible cases where that happened by excluding
+    // gas limit which are bellow a minimum threshold
```

Comment:
Pls remove magic values like `replace('0x', '0xd505accf')`

Also, the error handling feels a bit noisy and strange. 


Also, instead of `e.toString()` you mught want to extract the message if present